### PR TITLE
Added file extension icons + Included them on File component

### PR DESCRIPTION
**Pull request by piny4man**

### CODE REVIEW

File: src/components/Icon/stories.tsx [hunk 1]
```diff
@@ -26,6 +26,15 @@ storiesOf('Icon', module)
       <Icon icon="right" />
       <Icon icon="search" />
       <Icon icon="sent" />
-      <Icon icon="up" />
+      <Icon icon="pdf" />
```
Comment (mfmontenegro) [inline src/components/Icon/stories.tsx]: up icon missing

File: src/components/File/index.tsx [hunk 2]
```diff
@@ -1,20 +1,31 @@
...
 interface IProps extends IBaseProps {
   type: FileType
   name: string
+  transference?: Transference
 }
 
 const File: React.FC<IProps> = props => {
-  const { name, isHidden, className } = props
+  const { name, isHidden, className, type, transference } = props
+  const transferenceIcon = (): ReactNode => {
```
Comment (mfmontenegro) [inline src/components/File/index.tsx]: new line between declarative const and function

**Review by ghost — CHANGES_REQUESTED** [review]
Two minor comments

File: src/components/Icon/stories.tsx (hunk 1 above)
Comment (alberss) [reply src/components/Icon/stories.tsx]: Done

File: src/components/File/index.tsx (hunk 2 above)
Comment (alberss) [reply src/components/File/index.tsx]: Done

File: src/components/File/index.tsx [hunk 3]
```diff
@@ -1,20 +1,32 @@
...
   type: FileType
   name: string
+  isUserOwner?: boolean
 }
 
 const File: React.FC<IProps> = props => {
-  const { name, isHidden, className } = props
+  const { name, isHidden, className, type, isUserOwner } = props
+
+  const transferenceIcon = (): ReactNode => {
```
Comment (mfmontenegro) [inline src/components/File/index.tsx]: turn this into a const

**Review by ghost — DISMISSED** [review]
Turn function into a const

**Pull request closed without merging**